DependencyInjection/FDevsLocaleExtension.php

```diff
@@ -26,6 +26,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\XmlFileLoader($container, new FileLocator([__DIR__ . '/../Resources/config', realpath(dirname($refl->getFileName()) . '/Resources/config')]));
 
         $container->setParameter($this->getAlias() . '.allowed_locales', $config['allowed_locales']);
+        $container->setParameter($this->getAlias() . '.translator_extensions', $config['translator_extensions']);
```

Comment:
@andrey1s it seems that you use `f_devs_locale.translator_extensions` param only in the private service `f_devs_locale.twig_extension`. What do you think about removing translator_extensions param from global params bag and inject it via definition instead?